dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -179,6 +194,209 @@ public static Receiver internalReprOf(
         return receiver;
     }
 
+    public static Receiver internalReprOf(
+            AnnotationProvider provider, ExpressionTree receiverTree) {
+        return internalReprOf(provider, receiverTree, true);
+    }
+    /**
+     * We ignore operations such as widening and narrowing when computing the internal
```

Comment:
I followed the documentation style in the rest of this class.  This class is will be refactored after this pull request as we discussed.  When it's refactored, then the documentation will be fixed and the references to "internal representation" will be removed.